default/scripting/specials/specials.macros

```diff
@@ -13,7 +13,7 @@ CHANCE_OF_GUARD_1
             scope = Source
             activation = AND [
                 Turn high = 0
-                Random probability = 0.60
+                Random probability = 0.70
```

Comment:
As discussed with Matt, this will likely change to 100% when the AI team is ready for lots of ancient guardians. Morlic wanted the chances to stay as they are for now.